theano/gpuarray/multinomial.py

```diff
@@ -277,6 +273,7 @@ def make_node(self, pvals, unis, n):
 
         return Apply(self, [pvals, unis, as_scalar(n)], [out])
 
+    # Note by Amjad: I need to figure out how to convert from numpy dtypes to c datatypes
```

Comment:
I think you fround it and used it in the other op. Otherwise, tell me what you are looking for.